homeassistant/components/squeezebox_player/__init__.py

```diff
@@ -0,0 +1,54 @@
+"""Squeezebox Player integration."""
+from __future__ import annotations
+
+from aioslimproto import SlimServer
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EVENT_HOMEASSISTANT_STOP
+from homeassistant.core import Event, HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers import device_registry as dr
+
+from .const import DOMAIN
+
+PLATFORMS = ["media_player"]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up from a config entry."""
+    slimserver = SlimServer()
+    try:
+        await slimserver.start()
+    except Exception as exc:  # pylint: disable=broad-except
```

Comment:
Can we catch a more specific exception? What exception do we expect?

Reply:
It's low level socket operations. Anything can happen ;-)
I can add some more specific exceptions in the library later

Reply:
I've left out the try..except for now.